### PR TITLE
German docu updated

### PR DESCRIPTION
**Pull request by mischlrebl**

### CODE REVIEW

File: help/de-de.markdown [hunk 1]
```diff
@@ -31,6 +31,7 @@ comments: []
 <li><a href="#editor-highlight">Hervorheben</a></li>
 <li><a href="#editor-obfuscate">Unkenntlich machen</a></li>
 <li><a href="#editor-crop">Screenshot zuschneiden</a></li>
+<li><a href="#editor-enlarge">Screenshot vergrössern</a></li>
```
Comment (jklingen) [inline help/de-de.markdown:34]:
```suggestion
<li><a href="#editor-enlarge">Screenshot vergrößern</a></li>
```

File: help/de-de.markdown [hunk 2]
```diff
@@ -54,31 +55,27 @@ comments: []
 <h2>Erstellung eines Screenshots</h2>
 <p>
 Zum Erstellen eines Screenshots können Sie entweder die <kbd>Drucken</kbd>-Taste
-verwenden oder sie klicken das Greenshot-Icon im Systray mit der rechten Maustaste.<br>
+verwenden oder Sie klicken das Greenshot-Icon in der Taskleiste mit der rechten Maustaste.<br>
```
Comment (jklingen) [inline help/de-de.markdown:58]: I'm not sure about changing this to "Taskleiste" as it is less specific - it seems it is called "Infobereich" nowadays? What do you think about:
```suggestion
verwenden oder Sie klicken das Greenshot-Icon im Infobereich der Taskleiste mit der rechten Maustaste.<br>
```

File: help/de-de.markdown [hunk 3]
```diff
@@ -54,31 +55,27 @@ comments: []
...
-Mit Hilfe der <kbd>Leertaste</kbd> können sie vom Bereichsmodus in den
-<a href"#capture-window">Fenstermodus</a> wechseln (und umgekehrt).
+Mit Hilfe der <kbd>Leertaste</kbd> können Sie vom Bereichsmodus in den <a href="#capture-window">Fenstermodus</a> wechseln (und umgekehrt).
 </p>
 <p class="hint">
-Wenn Sie einen exakten Bereich abfotografieren, ist es eventuell einfacher, zuerst
-einen etwas größeren Bereich abzufotografieren und den Screenshot anschließend mit
-Hilfe des Bildeditors <a href="#editor-crop">zuzuschneiden</a>.
-</p>
+Wenn Sie einen exakten Bereich abfotografieren wollen, können Sie die exakte Position des Mauszeigers mithilfe der <kbd>Pfeil</kbd>-Tasten pixelgenau steuern. Das zusätzliche Drücken der <kbd>Strg</kbd>-Taste ändert die Position um 10 Pixel. <kbd>Enter</kbd> resp. <kbd>Return</kbd> übernimmt die Anfangs- respektive Endposition des Auswahlfensters. Mit der <kbd>Z</kbd>-Taste kann die Sichtbarkeit des Vergrösserungsglases umgeschaltet werden.
```
Comment (jklingen) [inline help/de-de.markdown:78]:
```suggestion
Wenn Sie einen exakten Bereich abfotografieren wollen, können Sie die exakte Position des Mauszeigers mithilfe der <kbd>Pfeil</kbd>-Tasten pixelgenau steuern. Das zusätzliche Drücken der <kbd>Strg</kbd>-Taste ändert die Position um 10 Pixel. <kbd>Enter</kbd> resp. <kbd>Return</kbd> übernimmt die Anfangs- respektive Endposition des Auswahlfensters. Mit der <kbd>Z</kbd>-Taste kann die Sichtbarkeit des Vergrößerungsglases umgeschaltet werden. 
```

File: help/de-de.markdown [hunk 4]
```diff
@@ -130,18 +127,20 @@ ergänzt werden. Es ist auch möglich, Teile des Screenshots hervorzuheben oder
 unkenntlich zu machen.
 </p>
 <p class="hint">
-Sie können Greenshots Bildeditor nicht nur für Screenshorts verwenden. Sie
+Sie können Greenshots Bildeditor nicht nur für Screenshots verwenden. Sie
 können auch Bilder aus Dateien oder aus der Zwischenablage zur Bearbeitung
-öffnen. Klicken Sie einfach mit der rechten Maustaste auf das Greenshot-Icon im
-Systray und wählen Sie <em>Bild aus Datei öffnen</em> bzw. <em>Bild aus
+öffnen. Klicken Sie einfach mit der rechten Maustaste auf das Greenshot-Icon in
+der Taskleiste und wählen Sie <em>Bild aus Datei öffnen</em> bzw. <em>Bild aus
```
Comment (jklingen) [inline help/de-de.markdown:133]:
```suggestion
öffnen. Klicken Sie einfach mit der rechten Maustaste auf das Greenshot-Icon im Infobereich
der Taskleiste und wählen Sie <em>Bild aus Datei öffnen</em> bzw. <em>Bild aus
```

File: help/de-de.markdown [hunk 5]
```diff
@@ -233,14 +233,21 @@ des Screenshots, den Sie behalten wollen. Sie können die Größe des ausgewähl
...
+<kbd>Strg</kbd> + <kbd>-</kbd> schneidet den Bereich sofort auf die vorhandenen Objekte zu.
 </p>
 <p class="hint">
 <em>Auto-Crop</em>: Wenn Sie einen einfarbigen Rahmen von Ihrem Screenshot entfernen möchten,
 wählen Sie einfach <em>Automatisch zuschneiden</em> aus dem Menü <em>Bearbeiten</em> und
 Greenshot wählt automatisch einen Bereich für den Zuschnitt aus.
 </p>
 
+<a name="editor-enlarge"></a>
+<h3>Screenshot vergrössern</h3>
```
Comment (jklingen) [inline help/de-de.markdown:246]:
```suggestion
<h3>Screenshot vergrößern</h3>
```

File: help/de-de.markdown [hunk 6]
```diff
@@ -233,14 +233,21 @@ des Screenshots, den Sie behalten wollen. Sie können die Größe des ausgewähl
...
+<p>
+Wenn Sie den Screenshot etwas vergrössern wollen, drücken Sie <kbd>Shift</kbd> + <kbd>+</kbd>, was auf allen vier Seiten je 25 Pixel hinzufügt.
```
Comment (jklingen) [inline help/de-de.markdown:248]:
```suggestion
Wenn Sie den Screenshot etwas vergrößern wollen, drücken Sie <kbd>Shift</kbd> + <kbd>+</kbd>, was auf allen vier Seiten je 25 Pixel hinzufügt.
```

File: help/de-de.markdown (hunk 2 above)
Comment (jklingen) [reply help/de-de.markdown:58]: btw: I also changed the english version to "notification area" instead of "systray"

Comment (mischlrebl) [reply help/de-de.markdown:58]: you are right, perfect

File: help/de-de.markdown (hunk 6 above)
Comment (mischlrebl) [reply help/de-de.markdown:248]: perfect. there is no ß in Switzerland, therefore they are all missing.... I thought to look at them, but my eyes are not trained for...

**Comment by mischlrebl** [comment]
You are welcome!

Oh, I would have done this as next. You should invest your effort where it is more important. Nevertheless, your English is for sure better than mine. But good to know that it is the master.

Btw: is there a way that I can collect all commits into one - or is it ok for you to have severals?

**Comment by Lakritzator** [comment]
Hi, we can squash the commits when we merge the PR, so don't bother 😉

**Comment by Lakritzator** [comment]
Thanks, very cool!!

**Pull request merged**